tix_clock/tix_clock.ino

```diff
@@ -37,3 +37,16 @@ void SetOutputPins(int outputPins[], int numberOfPins)
         pinMode(outputPins[i], OUTPUT);
     }
 }
+
+void GiveMeRandomNumber()
```

Comment:
This function returns an Integer, so it should be of int type:
int GiveMeRandomNumber

This function expects 2 arguments: The lower and upper limit:
int GiveMeRandomNumber(int lowerLimit, int upperLimit)

This function needs a prototype in line 13:
int GiveMeRandomNumber(int lowerLimit, int upperLimit);